RFC-008-release-versioning.md

```diff
@@ -0,0 +1,238 @@
+
+TGDP will use the Semantic Versioning convention for the versioning of individual Product Outputs (as at November 2021, this includes the template collection and the DocTools website package).
+
+> :triangular_flag_on_post: For the purposes of this RFC, "features" and "functionality" can refer to new **content**, such as additional sections or boilerplate text, added to doc templates.
+
+
+## Motivation
+
+Using the well-established [Semantic Versioning](https://semver.org/) framework will provide the following benefits:
+
+- It will demonstrate the project's progress to users and prospective users through publicly visible notifications.
+- It will set expectations to current users regarding the extent of change(s) contained in a newly-release version of a Product Output, as compared to the version the user currently has, including potentially breaking backward-compatibility.
+- It provides contributors to the project a common point of reference when collaborating, making the team more efficient.
+- It provides downstream integrators of TGDP's Product Outputs an easy way to identify if their own work products will need to be updated to support new features, or if there is a risk of compatibility issues.
+
+
+## Proposal
+
+In the course of preparing for a Release from the project:
+
+1. The PSC and Working Groups will decide on what functionality/features to target for it.
+1. Working Groups will work towards these goals during regular working sessions.
+1. Once the updates to a Product Output are **functionally complete**, meaning the initial implementation of new functionality has been completed, the Product Output can be tagged with a semantic target version number depending on changes introduced.
+1. The first such tag within a Release cycle should also carry the "-alpha" suffix.
+1. Further work during the release will focus on stabilizing the existing functionality and making it ready for general/production use.
+1. Once the Product Output is deemed ready, it will be tagged with the same semantic version number and the "-beta" suffix.
+1. At this point it should be released to a community of beta testers to actually use the Product Output. Feedback from this use should be actively solicited and captured.
+1. Provided there are no "blocker"-level issues issues remaining with the Product Output as the Release's target date approaches, the Release Manager will coordinate the completion of any final updates before signaling for them to be tagged with the same semantic version number, minus any suffix (indicating a gamma/final release).
+1. Once tagged, building and packaging the Product Outputs can commence.
+2. Following the gamma release of an Output, "patch" releases may be released in parallel to updated versions, as described in the section Semantic versioning addendum for documents below.
+
+## Semantic versioning addendum for documents
+

```

Comment:
Suggestion: This section is based on an external source document which I'm expecting to be getting updated by the documentation and developer community. (It currently has a bunch of review comments which are yet to be actioned).
As such, it may be helpful to reference the point of truth, and help readers understand whether this RFC content is the latest version or not.
```suggestion

Source: [Semantic versioning addendum for documents. Version 0.1-alpha.4. January 2020](https://docs.google.com/document/d/1gPr_OHoUkg5Wjj3UWYX78ORtTBArSJck4mWrKhWpfPA).

```

Reply:
I made a major mistake and accidentally merged this RFC in prematurely. I've added this suggestion to the body of the RFC.